java/src/main/java/com/google/appengine/tools/pipeline/impl/backend/CloudTasksTaskQueue.java

```diff
@@ -0,0 +1,118 @@
+package com.google.appengine.tools.pipeline.impl.backend;
+
+import com.google.appengine.tools.pipeline.impl.servlets.TaskHandler;
+import com.google.appengine.tools.pipeline.impl.tasks.PipelineTask;
+
+import com.google.cloud.tasks.v2.*;
+import com.google.protobuf.ByteString;
+import com.google.protobuf.Timestamp;
+import lombok.AllArgsConstructor;
+import javax.inject.Inject;
+import javax.inject.Provider;
+import java.net.URLEncoder;
+import java.nio.charset.StandardCharsets;
+import java.util.*;
+import java.util.stream.Collectors;
+
+import static com.google.appengine.tools.pipeline.impl.PipelineManager.DEFAULT_QUEUE_NAME;
+
+/**
+ * implementation of PipelineTaskQueue backed by Cloud Tasks
+ *
+ * TODO: what happens when tasks exists?
```

Comment:
if the name is the same you'll get an exception. The question is, should we ignore it or not

Reply:
yeah; previously it was a strongly typed exception for that case; now is generic at Java-client level, and we'd have to inspect the status code/message to tell